events/sr2017/kickstart.md

```diff
@@ -0,0 +1,49 @@
+---
+layout: news
+title: SR2017 Kickstart
+permalink: /events/sr2017/kickstart
+date: 2016-10-03
+---
+
+The venues for Student Robotics 2017 kickstart are:
+
+- University of Southampton
+- University of Bristol
+- The Kings Centre, Oxford
+
+Teams are welcome to attend any venue they wish. However; they should inform their local team co-ordinators which venue they will attend, as we will need to arrange for the robotics kits to be at the venue they choose, and some venues have vehicle access restrictions which will need to be arranged beforehand.
+
+Below is an incomplete table of venue-dependent information, which will be completed very soon.
+
+| Venue                     | How to get there                                     | Requires a laptop? | Nearby Food Locations | Finish Time |
+|---------------------------|------------------------------------------------------|--------------------|-----------------------|-------------|
+| University of Southampton | [Getting to campus][soton-campus-directions], and [on-campus directions][soton-directions] | No                 | [Map][soton-food-map] | 17:00 |
+| University of Bristol     | [Getting to campus][bris-campus-directions]. See [bristol directions](#bristol-directions) below for on-campus directions| Yes                |  [Map][bristol-food-map] | 16:00 |
+| The Kings Centre, Oxford  | [Download directions][oxford-directions]             | Yes                | [Map][oxford-food-map] | 16:00 |
+
+If teams cannot attend any of the venues, they should arrange for kit to be couriered to them. For some venues, teams will need to bring laptops or similar computing devices. internet access will be available. Teams will also be expected to make their own provisions for lunch.
+
+Here is a schedule for the day:
+
+|  Time  | Info |
+|--------|------|
+| 10:00  | Introductory presentation, where the game of SR2017 will be announced. |
+| 11:00  |Team kits will be handed out, and teams will take part in activities to get them acquainted with their kits. |
+| 12:30  | There will be a lunch break. Teams are expected to bring their own lunches or get them from food vendors near the venues. |
+|16:00 / 17:00| The activities will finish |
+
+
+## Bristol Directions
```

Comment:
Fixed in https://github.com/srobo/website/pull/28/commits/713f917f7792ba8555f7d0cd6685f78135f1a70f